Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:16
+
+COPY . /rdme-docker
```

Comment:
Not sure what the best practices are regarding directory stuff like this... feedback appreciated 😭

Reply:
It varies a lot AFAIK. I've seen `/app` before sometimes, that may be what we use in our main app. You could use `/rdme` if you want.